tests/test_app.py

```diff
@@ -88,6 +90,8 @@ def test_it_works_with_patch_p(self) -> None:
                         / entropy_validation: False]
                     / api_validation: NOT_AVAILABLE
                     / ml_validation: VALIDATED_KEY\n
+                    Scanning finished / Detected Credentials : 1\n
+                    Scanning finished / Detected Credentials : 0
```

Comment:
Possibly the double lines are confusing too.
Proposal: add argument to the function for printing result.
e.g. "Added diff scan finished" , "Removed diff..." Path scan finished"

Reply:
I'll remove `Scanning finished` phrase and just left `Detected Credentials` for each case(Diff scan, Normal scan).

Reply:
Alternatively:
Change `def scan` in `__main__.py` to return number of creds by checking something like `return len(credsweeper.credential_manager.get_credentials())`
https://github.com/Samsung/CredSweeper/blob/0d8ae1e8f112be0a5cbea2ab3f1037c90f2934ae/credsweeper/__main__.py#L151 

Then print can be done in `def main`, without passing any additional arguments:
https://github.com/Samsung/CredSweeper/blob/0d8ae1e8f112be0a5cbea2ab3f1037c90f2934ae/credsweeper/__main__.py#L187
https://github.com/Samsung/CredSweeper/blob/0d8ae1e8f112be0a5cbea2ab3f1037c90f2934ae/credsweeper/__main__.py#L193
https://github.com/Samsung/CredSweeper/blob/0d8ae1e8f112be0a5cbea2ab3f1037c90f2934ae/credsweeper/__main__.py#L197

Bonus: 
`def main` can also be used o measure execution time, and so exec time can also be added to the final output

Reply:
@meanrin 

Good suggestion.
I'll refer to it when I modify code.
Thank you.

Reply:
@meanrin linux may shrink the number.

Reply:
$ python -c "import sys; sys.exit(1223)"
$ echo $?
199

Reply:
> @meanrin linux may shrink the number.

Nah, I was not talking about exit code. Just a potential location to check the creds 


> $ python -c "import sys; sys.exit(1223)" $ echo $? 199

I think linux using only 1 byte for the exit code, so it just overflows anything greater than 255 

```
$ python -c "import sys; sys.exit(255)"
$ echo $?
255

$ python -c "import sys; sys.exit(256)"
$ echo $?
0

$ python -c "import sys; sys.exit(257)"
$ echo $?
1
```

Reply:
@meanrin unfortunately we have to support windows and mac. https://docs.python.org/3.4/library/os.html#os.EX_OK - Unix only. 
So, I think only 0 and 1 are valuable for exit codes and i propose use as is.
Ok - even credentials were found. Failure exit might brake some users scripts.
Failure - if CredSweeper cannot work totally. Probably, some errors may be accumulated like file access errors.

Reply:
I changed the code by [656a39a](https://github.com/Samsung/CredSweeper/pull/177/commits/656a39ac30d3e354dcbff17a55cf35bf39f45d6a).
Please check.